### DISCUSSION TITLE
有錢人都想變年輕?!

### DISCUSSION THREAD

**Discussion by youtinglin** [1101-GS4719-C/Python-Community-GS4719-C#1525] [General]
>###  真的可以返老還童嗎?

喝了什麼藥水或吃了什麼膠囊突然就進化或瞬間變年輕的劇情，感覺只會在科幻電影裡出現。
但隨著科技的進步，或許在未來的世界，這是會發生的事!

今年年初在頂尖學術期刊《Cell》上，發表了一篇關於表觀遺傳學（epigenetic）與哺乳類老化的研究文章。
此文章發表完後，引起了許多關注，在其他知名學術期刊如《Science》，與各大新聞媒體像是 CNN、TIME 雜誌都有報導。
內容指出科學家利用基因工程，去改變老鼠的表觀遺傳訊息，可加快老鼠的老化，亦或是透過生物學上的重新編程，讓已經老化的老鼠，重新變年輕。因此這項技術被認未有可能能預防人類老化，使我們可以延長壽命。

> ### 成為矽谷富豪們的熱門關鍵字
而為何會受到廣大的關注，不得不說起在矽谷的一家公司Altos Labs，此公司的目標是實現延長人類的壽命、甚至逆轉人類年齡抗老化的夢想。而這間公司的投資人大多是矽谷的富豪或各個公司的創辦人，且網羅許多傑出科學家。或許這就是體現年輕人想變有錢，有錢人想變年輕。

那麼大家對於這項技術的看法是什麼呢?
附上相關連結https://www.thebiologist.org/blog/230318-reverse-aging-rejuvenation-epigenetic-reprogram?ss_source=sscampaigns&ss_campaign_id=64862a6cb6062d6119861dfd&ss_email_id=6486602c7771ba258fa4b542&ss_campaign_name=Stay+in+the+Know.+And+More%EF%BD%9C%E5%85%AD%E6%9C%88%E7%AF%87%E3%80%82&ss_campaign_sent_date=2023-06-12T00%3A00%3A55Z

**Comment by Chiuhsianglin** [comment]
我看過一篇新聞在講說有一個美國科技富豪想要回春，所以和他70歲的爸爸、17歲的兒子三人在美國德州完成「三代換血」治療，外國傳媒報導指出，換血過程並非從一個人體內抽出血後，立即注射到另一人的體內。據知是由兒子提供1公升血液，通過機器分離成血漿、紅血球、白血球和血小板。富豪本人亦抽出1公升的血，抽血後隨即由靜脈輸入兒子的血漿。富豪的血液經同樣程序分離，之後輸入父親體內。我看到這篇新聞的時候也是感到非常不可思議，想說這樣真的可以幫助他變年輕嗎?但他的醫療團隊聲稱他的整體生理年齡減少5歲多，他現在擁有37歲的心臟、28歲的皮膚以及28歲的肺活量。我還是保持著懷疑的態度。
參考新聞:https://today.line.me/hk/v2/article/WBenDgE

**Comment by Miinteaa** [comment]
透過這篇文，我更加深了「時間對每個人都是公平的」這句話，看著每個人都想回到年輕的時刻，提醒了我真的應該趁自己還有時間好好把握機會，不要虛度光陰

**Comment by zhiron544** [comment]
我對於這項技術的看法是既期待又擔心。我期待它能夠幫助人類延長壽命，提高生活品質，也擔心它會造成社會和倫理上的問題，例如資源分配、人口控制、人性和價值觀的改變等。
